helper.php

```diff
@@ -537,4 +537,18 @@ public function deleteTags($tags, $namespace = '') {
 
         msg(sprintf($this->getLang("admin deleted"), count($tags), $numAffectedPages), 1);
     }
+
+    /**
+     * Updates tags with a new page name
+     *
+     * @param string $oldName
+     * @param string $newName
+     */
+    public function renamePage($oldName, $newName) {
+        $db = $this->getDb();
+        $res = $db->query('SELECT * FROM taggings WHERE pid = ?', $oldName);
+        if (!empty($db->res2arr($res))) {
```

Comment:
This check and select is not really needed, is it? You can simply issue the update statement below and it will affect either 1 or 0 rows.

Reply:
Right, I removed the redundant db query.